src/components/CellsWrapper.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { onMount, onDestroy } from 'svelte';
+  export let total = 0;
+  export let getCellProps = null;
+  export let component = null;
+
+  let columns = '';
+
+  switch (true) {
```

Comment:
Yeah, that's looks not very flexible.
I would consider some coefficient instead